apps/frontend/src/components/ui/ModerationChecklist.vue

```diff
@@ -646,16 +648,17 @@ For a brief rundown of how this works:
     {
       id: "gallery",
       navigate: `/${props.project.project_type}/${props.project.slug}/gallery`,
-      question: `Are the project's gallery images relevant?`,
-      shown: props.project.gallery.length > 0,
+      question: `Are this project's gallery images sufficient?`,
+      shown: true,
       options: [
         {
           name: "Not relevant",
+          shown: props.project.gallery.length > 0,
```

Comment:
I'm not completely sure what order they're shown in, but if I were you I would move Insufficient to be first and Not Relevant to be second because then the position in the checklist won't shift depending on whether the project has gallery images or not

Reply:
you're so right